### ISSUE TITLE
ListView items can't be invoked after scrolling the ListView on Windows 10 for Mobile build 14959?

### ISSUE THREAD

**Issue by naf456** [winjs/winjs#1654]
Ditto to the title.

I have found that on both of my Lumia 640's running latest fast ring build of Windows 10 (10.0.14959.1000), ListView items are failing / not recognising touch input after the listview has been scrolled (although strangely a small scroll of ~4px and it still works?).

You can replicate it with a simple test project with a sole list view.

I'm unsure if anyone knew about / cares, thus this issues report.

**Comment by cloudcrypt** [comment]
Are you using groupHeaders/grouping in your ListView?

**Comment by naf456** [comment]
No. A plain list view. It's was working on my laptop with a mouse, although my laptop at the time was on a lower build. Apologies for not replying; I don't spend a lot of time learning how to program anymore.

**Comment by lakichemole** [comment]
Hi, can you share the code,especially, do you use JavaScript only to populate your list, is it using virtualdatasource, do you use binding, template, how to you add action on item?

**Comment by VictorOdukoya** [comment]
Hello, I have the same issue. I can scroll trough my list with my mouse without a problem, but if I use the touchscreen and scroll trough it (or touch hold an item and move my finger out of the window) the hover effect stays on the item and the items can't be invoked.

I created the template dynamically.

**Comment by mfycan** [comment]
Thankfully, it seems they are aware of the issue. Already got e-mails complaining that they can't select any list item on my app.

**Comment by drogerie21** [comment]
hi, i have the same Problem, but on Windows 10 version 1703 (OSBuild 15063.413) - no insider preview. On Touch Devices as soon i have to scroll in a listview, the listview does not react to touch events. This does not happen with windows 10 versions prior to 1703 (e.g. version 1607). so i assume this problem comes up with the creators update?
